Providers/nxOMSAutomationWorker/MSFT_nxOMSAutomationWorkerResource.schema.mof

```diff
@@ -1,7 +1,5 @@
-[ClassVersion("1.2.0")]
+[ClassVersion("1.5.0")]
 class MSFT_nxOMSAutomationWorkerResource : OMI_BaseResource
 {
-    [Key] string WorkspaceId;
-    [Write] boolean Enabled;
-    [Write] string AzureDnsAgentSvcZone;
+    [Key] string ResourceSettings;
```

Comment:
So you still have to restart the OMI service to apply this module correctly? If this is true, I don't think this change should be made yet, unless you have an acceptable way to restart the OMI service.